dlt-connector/src/graphql/input/TransactionInput.ts

```diff
@@ -0,0 +1,21 @@
+// https://www.npmjs.com/package/@apollo/protobufjs
+
+import { Decimal } from 'decimal.js-light'
+import { TransactionType } from '../enum/TransactionType'
+import { InputType, Field } from 'type-graphql'
+
+@InputType()
+export class TransactionInput {
```

Comment:
In graphql you cannot use ObjectType as Input:
https://graphql.org/learn/schema/#input-types
here is stated why: 
https://spec.graphql.org/October2016/#sec-Input-Objects
If you look in the backend code you will notice that the model Objects are only used as output and for input @ArgsType and @InputTypes in folder /arg are used.

Why I use input rather than arg?
Input is used for mutations, args for querys and I don't know if I even will have a single query.